src/crimson/osd/ops_executer.h

```diff
@@ -519,14 +521,15 @@ OpsExecuter::flush_changes_n_do_ops_effects(
       std::move(txn),
       std::move(obc),
       std::move(*osd_op_params),
-      std::move(log_entries));
+      std::move(log_entries),
+      cloning_ctx
+	? std::move(cloning_ctx->clone_obc)
```

Comment:
I see failure in the latest main:
```
crimson/osd/ops_executer.h:526:27: runtime error: member access within null pointer of type 'struct CloningContext'
```

To reproduce:
```
$MGR=0 MON=1 OSD=1 MDS=0 RGW=0 ceph/src/vstart.sh -n -x --without-dashboard --crimson --nodaemon --redirect-output --seastore
$./bin/ceph osd pool create test-pool 32 32 && ./bin/ceph osd pool set test-pool size 1 --yes-i-really-mean-it && ./bin/rbd create test-pool/test-img --size 50G --image-format=2 --rbd_default_features=3
```

Reply:
Ok, will get into it.

Reply:
@cyx1231st I can't reproduce this error locally, you are running the main branch, right?

On the other hand, the error looks odd, the codes around `ops_executer.h:526` are as follows:
https://github.com/ceph/ceph/blob/78b4fdf4f580e37367798d07298a89266f100d0a/src/crimson/osd/ops_executer.h#L520-L528

It seems that the access to `cloning_ctx->clone_obc` is protected by the check `cloning_ctx`, which really confuses me.

Reply:
see https://github.com/ceph/ceph/pull/60913 or https://github.com/ceph/ceph/pull/60914